src/category_theory/limits/shapes/types.lean

```diff
@@ -61,13 +62,21 @@ def terminal_limit_cone : limits.limit_cone (functor.empty (Type u)) :=
     π := by tidy, },
   is_limit := by tidy, }
 
+/-- The terminal object in `Type u` is `punit`. -/
+noncomputable def terminal_iso : ⊤_ (Type u) ≅ punit :=
```

Comment:
Do we want lemmas describing how this composes with the projections? I imagine in one direction it's true by subsingleton and in the other it's ext; subsingleton, but it might be useful to have them as simp lemmas anyway?